lib/mockbuilder.py

```diff
@@ -69,8 +69,10 @@ def build(self, package):
         LOG.info("%s: Starting build process" % package.name)
         self._build_srpm(package)
         self._install_external_dependencies(package)
-        cmd = (self.common_mock_args + " --rebuild %s --no-clean --resultdir=%s"
-               % (self.build_dir + "/*.rpm", self.build_dir))
+        cmd = (self.common_mock_args
+               + " --rebuild %s --no-clean --resultdir=%s" % (
+                   self.build_dir + "/*.rpm", self.build_dir)
+               + package.get_spec_macros())
```

Comment:
minor: I do not understand why mix %s and + . Why not "--rebuild %s --no-clean --resultdir=%s %s" % (self.build_dir + "/*.rpm", self.build_dir, package.get_spec_madros()) ?